yb-voyager/cmd/importData.go

```diff
@@ -992,6 +993,10 @@ func markTaskDone(task *SplitFileImportTask) {
 	if err != nil {
 		utils.ErrExit("rename %q => %q: %s", inProgressFilePath, doneFilePath, err)
 	}
+
+	if !truncateSplits {
+		return
+	}
 	err = os.Truncate(doneFilePath, 0)
 	if err != nil {
 		log.Warnf("truncate file %q: %s", doneFilePath, err)
```

Comment:
Returning like this may cause bugs in future. Consider a case where we add some more code after truncate. You can refactor it like this:
```
if truncateSplits {
  err = os.Truncate(...)
  ...
}
```

Reply:
ok, just wanted to save the identatiton for the code here.